minecraft/src/main/java/net/fabricmc/loader/impl/game/minecraft/MinecraftGameProvider.java

```diff
@@ -318,10 +320,22 @@ public void initialize(FabricLauncher launcher) {
 				obfJars.put("realms", realmsJar);
 			}
 
+			String sourceNamespace = "official";
+
+			try {
+				VersionPredicate mergedVersionPredicate = VersionPredicate.parse(">=1.3");
+
+				if (!mergedVersionPredicate.test(Version.parse(getNormalizedGameVersion()))) {
+					sourceNamespace = envType == EnvType.CLIENT ? "clientOfficial" : "serverOfficial";
```

Comment:
(Leaving a comment, as requested by @thecatcore) Treating an enum value as a boolean feels wrong, even if it will always realistically be one or the other.

Reply:
I think in this case it's fine, even if strictly something like a switch statement would be better. I don't see that enum getting another value.